src/pages/Unlock/index.tsx

```diff
@@ -1,48 +1,61 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { useGetAccount, useGetNetworkConfig } from "@multiversx/sdk-dapp/hooks";
 import { NativeAuthConfigType } from "@multiversx/sdk-dapp/types";
 import { logout } from "@multiversx/sdk-dapp/utils/logout";
+import { useWallet } from "@solana/wallet-adapter-react";
 import { WalletMultiButton } from "@solana/wallet-adapter-react-ui";
+import { ArrowBigLeft } from "lucide-react";
 import { useLocation, useNavigate } from "react-router-dom";
 import { AuthRedirectWrapper, ExtensionLoginButton, WalletConnectLoginButton, WebWalletLoginButton, LedgerLoginButton } from "components";
 import { walletConnectV2ProjectId } from "config";
 import { Button } from "libComponents/Button";
 import { getApi } from "libs/utils";
-import { routeNames } from "routes";
-import { ArrowBigLeft } from "lucide-react";
+// import { routeNames } from "routes";
+
+// // find a route name based on a pathname that comes in via React Router Link params
+// function getRouteNameBasedOnPathNameParam(pathname: string) {
+//   const matchPathnameToRouteName = Object.keys(routeNames).find((i: string) => {
+//     return (routeNames as any)[i] === pathname;
+//   });
 
-// find a route name based on a pathname that comes in via React Router Link params
-function getRouteNameBasedOnPathNameParam(pathname: string) {
-  const matchPathnameToRouteName = Object.keys(routeNames).find((i: string) => {
-    return (routeNames as any)[i] === pathname;
-  });
+//   if (matchPathnameToRouteName && matchPathnameToRouteName !== "home") {

```

Comment:
Comments left intentionally?

Reply:
some legacy comments, leaving in case it's relevant to some boundry case issue